src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -94,23 +128,34 @@ public LocalDateTime getDateInSpecificTimeZone(String dateInString, String zone)
      * OffsetDateTime is recommended to use for storing date values in a database.
      */
     public OffsetDateTime offsetDateTime(LocalDateTime localTime) {
-        return OffsetDateTime.now();
+        return OffsetDateTime.of(localTime, ZoneId.systemDefault().getRules().getOffset(localTime));
```

Comment:
let's move the offset for Ukraine `+02:00` to the constant field and use it it the solution